src/components/CheckboxList.js

```diff
@@ -0,0 +1,54 @@
+import { Typography } from "@mui/material";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import DeleteIcon from "@mui/icons-material/Delete";
+import IconButton from "@mui/material/IconButton";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
+
+export default function CheckboxList({ addTask, deleteTask }) {
```

Comment:
`addTask` sounds like a name for function but it is actually an array of task. In general we want more meaningful names for props, functions and constants.

Reply:
Done (changed to "Tasks")